apps/examples/lcd_test/example_lcd.c

```diff
@@ -446,8 +510,15 @@ int lcd_test_main(int argc, char *argv[])
 	}
 	test_fps();
 	close(fd);
+errout:
```

Comment:
it will not always errout case
this line could execute in OK case also.
change the name errout

Reply:
fixed